tests/test_single_topology_v3.py

```diff
@@ -288,7 +288,7 @@ def setup_arbitary_transformation():
         mol_b = mols["57"]
 
         mcs_threshold = 0.75
-        res = atom_mapping.mcs_map(mol_a, mol_b, mcs_threshold)
+        res = atom_mapping.mcs(mol_a, mol_b, mcs_threshold)
```

Comment:
same with 0.75 here ^

Reply:
Modified all instances of `mcs_threshold = 0.75` to `mcs_threshold = 2.0` in https://github.com/proteneer/timemachine/pull/812/commits/2b91826c31d69235fdd90b5b9c3b31cc82533bf4